### DISCUSSION TITLE
Project/Business Name

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#22] [Q&A] [answered]
Since Knowledge for Good is already an established nonprofit and the website knowledgeforgood.com is unavailable (granted we could easily go with a different ending ) do we want to consider alternate names or would we be able to establish as a co-op with the same name?

**Comment by avidrucker** [comment]
I would love to brainstorm more names... Especially as this is something we are co-creating. I am also happy to continue to use KFG if it brings you joy, in a different context. For example, KFG + seekanddefine = 💗

**Comment by seekanddefine** [comment, marked as answer]
We are officially Knowledge for Good!
